HealthGPT/HealthGPT/HealthGPTView.swift

```diff
@@ -6,60 +6,56 @@
 // SPDX-License-Identifier: MIT
 //
 
-import HealthKit
-import OpenAI
 import SpeziFHIR
-import SpeziSecureStorage
+import SpeziOpenAI
 import SwiftUI
 
 struct HealthGPTView: View {
-    @AppStorage(StorageKeys.openAIModel) var openAIModel = Model.gpt3_5Turbo
-    @EnvironmentObject var secureStorage: SecureStorage<FHIR>
-    @State private var messages: [Message] = []
-
-    @State private var showAlert = false
-    @State private var alertText = ""
-
-    @StateObject private var messageManager = MessageManager()
-
+    @AppStorage(StorageKeys.onboardingFlowComplete) var completedOnboardingFlow = false
+    @EnvironmentObject private var openAPIComponent: OpenAIComponent<FHIR>
+    @EnvironmentObject private var healthDataInterpreter: HealthDataInterpreter<FHIR>
+    @State private var showSettings = false
+    
```

Comment:
We normally add two spaces between the different element types within a type. E.g., here between stored properties and computed properties:
```suggestion
    
    
```